schemas/ships.json

```diff
@@ -917,197 +904,80 @@
-                                            "description": "The type of other reward gained.",
-                                            "type": "string",
-                                            "default": "",
-                                            "examples": [
-                                              "ship xp"
-                                            ]
-                                          },
-                                          "value": {
-                                            "$id": "#/properties/tiers/items/anyOf/0/properties/levels/items/anyOf/0/properties/scrap_details/properties/rewards/properties/others/items/anyOf/0/properties/value",
-                                            "title": "The value schema",
-                                            "description": "The amount of this other reward gained.",
-                                            "type": "integer",
-                                            "default": 0,
-                                            "examples": [
-                                              25000
-                                            ]
-                                          }
-                                        }
-                                      }
-                                    ]
-                                  }
-                                },
-                                "resources": {
-                                  "$id": "#/properties/tiers/items/anyOf/0/properties/levels/items/anyOf/0/properties/scrap_details/properties/rewards/properties/resources",
-                                  "title": "The resources schema",
-                                  "description": "List of all resources gained.",
-                                  "type": "array",
-                                  "additionalItems": true,
-                                  "default": [],
-                                  "items": {
-                                    "$id": "#/properties/tiers/items/anyOf/0/properties/levels/items/anyOf/0/properties/scrap_details/properties/rewards/properties/resources/items",
-                                    "anyOf": [
-                                      {
-                                        "$id": "#/properties/tiers/items/anyOf/0/properties/levels/items/anyOf/0/properties/scrap_details/properties/rewards/properties/resources/items/anyOf/0",
-                                        "title": "The first anyOf schema",
-                                        "description": "Single resource instance.",
-                                        "required": [
-                                          "type",
-                                          "value"
+                                        "examples": [
+                                          3
+                                        ]
+                                      },
+                                      "rarity": {
```

Comment:
Is the rarity optional here because not all resources have a rarity (like iso and lat)?

Reply:
For rewards are grade and rarity always optional.